tty.go

```diff
@@ -24,14 +25,20 @@ func randomPort() int {
 
 // StartTTY starts the ttyd process on the given port.
 func StartTTY(port int) *exec.Cmd {
-	//nolint:gosec
-	cmd := exec.Command(
-		"ttyd", fmt.Sprintf("--port=%d", port),
+	args := []string{
+		fmt.Sprintf("--port=%d", port),
 		"-t", "rendererType=canvas",
 		"-t", "disableResizeOverlay=true",
 		"-t", "cursorBlink=true",
 		"-t", "customGlyphs=true",
-		"bash", "--login",
-	)
+	}
+	switch runtime.GOOS {
+	case "windows":
+		args = append(args, "cmd.exe")
+	default:
+		args = append(args, "bash", "--login")
```

Comment:
after #55 we can probably default to just `sh`

Reply:
Sounds like a plan!